Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -580,6 +605,10 @@ - (void)uploadAvatarToService:(NSData *)avatarData
                     }];
             }
             failure:^(NSURLSessionDataTask *task, NSError *error) {
+                if (task.response != nil) {
```

Comment:
What is implied by this if statement? Can you leave a comment?

Reply:
Done. Please check out the comment.